mobile/src/navigation/app/components/become-mentor/styles.ts

```diff
@@ -0,0 +1,27 @@
+import { StyleSheet } from 'react-native';
+
+import { AppColor } from '~/common/enums/enums';
+
+const styles = StyleSheet.create({
+  footer: {
+    alignItems: 'center',
+    justifyContent: 'flex-end',
+    height: 140,
+    marginTop: 130,
+    marginBottom: 30,
+    marginHorizontal: 24,
+    paddingHorizontal: 20,
+    paddingBottom: 20,
+    backgroundColor: AppColor.BACKGROUND.GRAY_400,
+    borderRadius: 14,
+  },
+  footerImage: {
+    marginLeft: 20,
+    marginBottom: 10,
+    resizeMode: 'contain',
```

Comment:
It's better to avoid contain, because if height/width is different from original image properties, it won't look good in empty spaces
```suggestion
```